src/opencast.js

```diff
@@ -384,6 +427,22 @@ export class Opencast {
     return this.#login === true;
   }
 
+  getLoginUsername(){
+    return this.#login?.username;
+  }
+
+  getLoginPassword(){
+    return this.#login?.password;
+  }
```

Comment:
I don't quite get the purpose of these methods. They are only used in `header` (which can hopefully be removed anyway since it contains duplicate logic) and it's not like `this.getLoginUsername()` is shorter than `this.#login?.username`...